week-1/Homework/extra/2-piping.js

```diff
@@ -16,26 +16,34 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  let decimal_allowed = a + b;
+  let precision = decimal_allowed.toFixed(1) / 1;
```

Comment:
One interesting question is _where_ to do the rounding here.

In general, we try to round numbers as late as we can, ideally just when we're formatting, rather than intermediate results. If you imagine:
```javascript
multiply(4, add(1.2, 1.05))
```
if we round in `add`, the answer is `4`; if don't, the answer is `5` - by leaving the rounding to the very end, we get more accurate results.